shark-graph/src/main/java/shark/internal/Tuples.kt

```diff
@@ -0,0 +1,69 @@
+package shark.internal
+
+/**
+ * Replacement to a traditional Pair<Long, Object> that doesn't box long.
+ * Class is purposefully not made as a `data class` to decrease memory footprint of the object.
+ */
+class LongPair<out B>(
```

Comment:
class should be internal, you don't want to pollute everyone's classpath

Reply:
`PathFinder` (from `shark` module) uses `LongLongScatterMap` (from `shark-graph`) which uses `LongLongPair` (also from `shark-graph`). Can't make `LongLongPair` internal :(

Reply:
Copy paste will do until we have internal across modules.